web/app/signin/components/mail-and-password-auth.tsx

```diff
@@ -124,7 +125,12 @@ export default function MailAndPasswordAuth({ isInvite, allowRegistration }: Mai
     <div className='mb-3'>
       <label htmlFor="password" className="my-2 flex items-center justify-between">
         <span className='system-md-semibold text-text-secondary'>{t('login.password')}</span>
-        <Link href={`/reset-password?${searchParams.toString()}`} className='system-xs-regular text-components-button-secondary-accent-text'>
+        <Link
+          href={`/reset-password?${searchParams.toString()}`}
+          className={`system-xs-regular ${isEmailSetUp ? 'text-components-button-secondary-accent-text' : 'text-components-button-secondary-accent-text-disabled pointer-events-none'}`}
+          tabIndex={isEmailSetUp ? 0 : -1}
+          aria-disabled={!isEmailSetUp}
+        >
```

Comment:
## Reference
https://stackoverflow.com/questions/73555618/how-can-i-disable-link-href-in-next-js-on-various-conditions